File: compiler/testData/loadJava/compiledJava/annotations/AnnotatedParameterInInnerClassConstructor.java
package test;

class AnnotatedParameterInInnerClassConstructor {

    public @interface Anno {
        String konstue();
    }

    class Inner {
        Inner(@Anno("a") String a , @Anno("b")  String b) {}
    }

    class InnerGeneric<T> {
        InnerGeneric(@Anno("a") String a , @Anno("b")  String b) {}
    }
}

File: compiler/testData/asJava/lightClasses/lightClassByFqName/AnnotatedParameterInInnerClassConstructor.kt
// test.AnnotatedParameterInInnerClassConstructor
package test

annotation class Anno(konst x: String)

class AnnotatedParameterInInnerClassConstructor {

    inner class Inner(@Anno("a") a: String, @Anno("b") b: String) {

    }

    inner class InnerGeneric<T>(@Anno("a") a: T, @Anno("b") b: String) {

    }
}

// FIR_COMPARISON